### ISSUE TITLE
Not find my dishwasher

### ISSUE THREAD

**Issue by crusader85** [ekutner/home-connect-hass#375]
As the main integration from HomeConnect stop working with my dishwasher, I found this integration. But this integration doesn't find my dishwasher. All of the other devices working as expected.

**Comment by ekutner** [comment]
Please read the README and specifically the FAQ before opening issues. You should remove your appliance from the HC service and then add it back.

**Comment by crusader85** [comment]
Das habe ich ja bereits getan, es funktioniert dennoch nicht.

**Comment by ekutner** [comment]
Please describe exactly what you did to remove and add the dishwasher back to Home Connect. Do you see the it in your Home Connect mobile app when **wifi is turned off** on your phone?

**Comment by kenni** [comment]
Ok, I've had the same issue for the past couple of weeks. Suddently the dishwasher wasn't provided by the integration. I realized that the device was only controllable from Home Connect when connected to the local LAN. Removing the device and re-adding it to Home Connect did not resolve the issue. Only after *power cycling* the dishwasher, the cloud connection was re-established. I had restarted the device many times using the power button on the dishwasher, but this wasn't sufficient. I've sent a pull-request to update the FAQ to explicitly state that it needs to be a full power cycle and not just a restart by turning the device off and on.
